### PR TITLE
[Improve][Hive] udpate hive3 version

### PR DESCRIPTION
**Pull request by EricJoy2048**
### Purpose of this pull request

Tests show that the Hive (Hive3) connector has better compatibility with `hive-exec-3.1.3.jar`

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: docs/en/connector-v2/source/Hive.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Read data from Hive.
 
 In order to use this connector, You must ensure your spark/flink cluster already integrated hive. The tested hive version is 2.3.9.
```
Comment (TaoZex) [inline docs/en/connector-v2/source/Hive.md:11]: Does this line also need to be modified？

Comment (EricJoy2048) [reply docs/en/connector-v2/source/Hive.md:11]: > Does this line also need to be modified？

Yes, I will update it in the next two days.

Comment (EricJoy2048) [reply docs/en/connector-v2/source/Hive.md:11]: I haven't tested hive version 3.1.3 on Flink and Spark, so it should remain unchanged here.

**Review by TaoZex — APPROVED** [review]
+1

**Review by Hisoka-X — APPROVED** [review]

**Review by hailin0 — APPROVED** [review]

**Pull request merged**